{"uuid": "047189c4-a4ab-4699-bb05-10ab9ab7a5d7", "children": ["c65b9eda-3197-4011-a9c6-64df333442c8", "9ca3386f-4e4c-463d-989b-8c7e8d843fc9", "4cee2dad-24bc-4443-a9e1-f30c2617969e", "072ac489-df9e-4794-a820-9142ed08856d", "f0bbbef3-b81e-4523-9e59-08a41b3abea4"], "befores": [{"name": "get_job_details", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.bbcollection.ro', port=443): Max retries exceeded with url: /cariera.html (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x7fbbb0f62c80>: Failed to resolve 'www.bbcollection.ro' ([Errno -3] Temporary failure in name resolution)\"))\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/Test_API_Suite/test_bbcollection.py\", line 14, in get_job_details\n    scraper_data = bbcollectionScraper().return_data()\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/sites/bbcollection.py\", line 53, in return_data\n    self.get_response()\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/sites/bbcollection.py\", line 26, in get_response\n    self.get_content(self.url)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/sites/website_scraper_bs4.py\", line 34, in get_content\n    response = requests.get(url, headers=self.DEFAULT_HEADERS, verify=False)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests/api.py\", line 73, in get\n    return request(\"get\", url, params=params, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests/sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests/sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests/adapters.py\", line 519, in send\n    raise ConnectionError(e, request=request)\n"}, "start": 1707248667432, "stop": 1707248677449}], "start": 1707248667432, "stop": 1707248677973}